pkgs/applications/blockchains/gaiad/default.nix

```
@@ -0,0 +1,34 @@
+{ lib, stdenv, buildGoModule, fetchzip, nixosTests }:
+
+let
+  bins = [
+    "gaiad"
+  ];
+
+in
+buildGoModule rec {
+  pname = "gaia";
```

we have `pkgs.gaiad`, `pname = "gaia";`, and `bin/gaiad`, should they all just be `gaiad`?
```suggestion
  pname = "gaiad";
```